sdk/src/AbstractMain.cpp

```diff
@@ -238,7 +275,26 @@ int AbstractMain::run(int in_argc, char** in_argv)
                getProgramId(),
                options.getScratchPath())));
    }
+   // Ensure logging directory path exists and is configured correctly.
+   int ret_log_dir = configureLoggingDir(options.getLoggingDir(), serverUser, options.useUnprivilegedMode());
+   if (ret_log_dir != 0)
+      return ret_log_dir;
 
+   if (options.getEnableDebugLogging())
+   {
+      options.getLoggingDir().changeFileMode("ALL_READ_WRITE_EXECUTE");
+      options.getLoggingDir().changeOwnership(serverUser);
+      std::string logName = "rstudio-launcher-debug";
```

Comment:
```suggestion
      std::string logName = getProgramId() + "-debug";
```

We want to use the name of the plugin in the log file name, not `rstudio-launcher`. Since this is the plugin logs, not the actual Launcher.